App/components/DatabaseComponent.js

```diff
@@ -1,17 +1,69 @@
 import React from "react";
-import {
-  ScrollView,
-  StyleSheet,
-  Text,
-  View,
-  TouchableOpacity,
-} from "react-native";
+import { ScrollView, Image, StyleSheet, Text, View, TouchableOpacity } from "react-native";
+//import Items from "../components/Items.js";
 import Constants from "expo-constants";
 
 import * as SQLite from "expo-sqlite";
 
 const db = SQLite.openDatabase("Grocery_Items.db");
 
+export default class DataBaseComponent extends React.Component {
+  state = {
+    name: null,
+  };
+
+  //if there is not a table in the database, then create one
+  componentDidMount() {
+    db.transaction((tx) => {
+      tx.executeSql("create table if not exists items (id integer primary key not null, name text, expirationDate date, price text);");
+    });
+  }
+
+  //adding the items to the database
+  add(name, expirationDate, price) {
+    // is name empty?
+    if (name === null || name === "") {
+      return false;
+    }
+
+    //attributes line up with the quesiton marks in the corresponding order
+    db.transaction((tx) => {
+      tx.executeSql("insert into items (name, expirationDate, price) values (?, ?, ?)", [name, expirationDate, price]);
+      tx.executeSql("select * from items", [], (_, { rows }) => console.log(JSON.stringify(rows)));
+    }, null);
+  }
+
+  render() {
+    return (
+      <View style={styles.container}>
+        <Text style={styles.heading}>My List</Text>
+        <Text style={styles.headingDiscp}>This is your current working shopping list!</Text>
+
+        <TouchableOpacity
+          style={styles.sortBtn}
```

Comment:
Since we are working with delete I think renaming sortBtn style to something like deleteBtn style might be more appropriate since sortBtn kind of misleading.

Reply:
thanks for point that out! i was working on sorting before this and i forgot to change the name